src/toast/_toast.scss

```diff
@@ -0,0 +1,20 @@
+.toast {
+  position: fixed;
+  bottom: 0;
+  z-index: 2;
+
+  width: 100vw;
+
+  padding: 17px 0;
+
+  text-align: center;
+
+  &--danger {
+    color: red;
```

Comment:
Let's make it CSS variable

`--toast-danger-text-color`
`--toast-danger-bg`

`--toast-warning-text-color`
`--toast-warning-bg`

`--toast-success-text-color`
`--toast-success-bg`